nodes/nomos-executor/config.yaml

```diff
@@ -70,6 +70,7 @@ da_network:
         seen_message_ttl: "3600.0"
       subnets_settings:
         num_of_subnets: 20
+        retry_limit: 5
```

Comment:
This will probably be hard-coded or come from a config in the genesis block. We can let it in config for now tho.